x-pack/plugins/infra/server/lib/alerting/log_threshold/log_threshold_executor.ts

```diff
@@ -463,6 +502,15 @@ export const updateAlert: AlertUpdater = (alert, state, actions) => {
   });
 };
 
+const getViewInAppUrl = (kibanaBaseUrl: string | undefined, indexedStartedAt: string) => {
+  // Need to extract the kibana base path if exists (server.basePath) to concatenate it correctly using the URL Api.
+  const kibanaBasePath = kibanaBaseUrl ? new URL(kibanaBaseUrl).pathname : '';
```

Comment:
@fkanout Since view in app url is constructed based on `kibanaBaseUrl`, do you think that we should explain this in the `viewInAppUrlActionVariableDescription`? I don't know much about `kibanaBaseUrl`, but is it something defined by the user? I remember a discussion we had with @simianhacker and @vinaychandrasekhar about making it clear to the user how this url is constructed, that's why I am bringing it up here.

Reply:
@mgiota, we could add an explanation. But for me, I don't think we need to. As it's very technical it could backfire with confusion depending on the user using it. 
`kibanaBaseUrl` is the domain name where Kibana is hosted. e.g. `https://mykibana.com` + `kibanaBasePath`  `/dev` --> `https://mykibana.com/dev`. The way how it's implemented If the `kibanaBaseUrl` is provided we build the full URL (from what I got from @simianhacker, on our Cloud solution, it's always provided, and most of the other cases too). But if not, we build a relative one.